css/media.css

```diff
@@ -0,0 +1,272 @@
+@media screen and (min-width: 768px) {
+  .headline {
+    padding: 0 24px;
+  }
+
+  .headline img {
+    display: none;
+  }
+
+  .nav {
+    list-style: none;
+    display: flex;
+    gap: 0 32px;
+  }
+
+  .nav a {
+    text-decoration: none;
+    color: #fff;
+    font-size: 15px;
+    font-weight: 600;
+    font-family: "Inter", sans-serif;
+  }
+
+  .nav li:last-child {
+    padding-right: 100px;
+  }
+
+  .nav li a:hover {
+    color: #e05304;
+    transition: all 0.3s;
+  }
+
+  .contact-icon {
+    background: url(../Images/icons/contact-icon.svg) no-repeat center center/cover;
+    width: 22px;
+    height: 18px;
+    display: block;
+  }
+
+  .headline_section {
+    overflow: none;
+    display: flex;
+    background-image: url(../Images/Illustration-header.svg);
+    background-position: right top;
+    background-repeat: no-repeat;
+    background-size: 440px 550px;
+    background-color: #1c1a19;
+    flex-direction: row-reverse;
+    height: 100vh;
+  }
+
+  .headline_section :nth-child(1) {
+    margin-top: auto;
+  }
+
+  .headline_section :nth-child(2) {
+    margin-top: auto;
+    flex-basis: 5px;
+  }
+
+  .image-icons-header {
+    display: flex;
+    flex-direction: column-reverse;
+    gap: 20px;
+    padding: 16% 2% 251px 2%;
+  }
+
+  .headline_section h1 {
+    font-size: 72px;
+    line-height: 81px;
+  }
+
+  .p {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    padding-bottom: 251px;
+    width: 39.3%;
+    color: #f4f5f7;
+  }
+
+  .recent {
+    width: 100%;
+    padding: 80px;
+    color: #091e42;
+  }
+
+  .line {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 40px;
+    justify-content: center;
+    position: relative;
+    margin-bottom: 30px;
+    color: #091e42;
+  }
+
+  .line h2 {
+    float: left;
+    top: 70px;
+    left: 20px;
+  }
+
+  .line hr {
+    text-align: center;
+    height: 2px;
+    width: 70%;
+    background-color: #091e42;
+    float: right;
+    top: 100px;
+  }
+
+  .img-txt {
+    display: flex;
+    flex-direction: row;
+    gap: 15px;
+  }
+
+  .card-section {
+    width: 100%;
+    display: grid;
+    grid-template-rows: 300px 300px 300px;
+    grid-template-columns: 366px 366px 366px;
+    align-items: center;
+    justify-content: center;
+    row-gap: 120px;
+    margin-bottom: 0;
+  }
+
+  .background-graphic {
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    background: url(../Images/desktop-about.png) no-repeat;
+    background-size: 700px 359px;
+    background-position: right top;
+  }
+
+  .horizontal-seperator-on-dark {
+    margin-top: 157px;
+    width: 80%;
+    align-self: center;
+  }
+
+  #about-section {
```

Comment:
- [ ] Please I highly recommend working on the left and right `margins`  of the `about-section` to fit the full `width` of the screen. 😄 

_**YOUR DESIGN**_


![Screenshot (191)](https://user-images.githubusercontent.com/78104544/229261645-12d9cde7-92b2-4698-a06b-e1912560f366.jpg)